src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1503,7 +1485,17 @@ internal bool GetExternAliasTarget(string aliasName, out NamespaceSymbol @namesp
         /// </summary>
         internal new NamedTypeSymbol? ScriptClass
         {
-            get { return _scriptClass.Value; }
+            get
+            {
+                // _scriptClass is allowed to be null, thus why a bool is used to track initialization
```

Comment:
❓ How often is this actually needed? If rarely, I'd prefer this:

```
private StrongBox<NamedTypeSymbol?>? _scriptClass;

return InterlockedOperations.Initialize(ref _scriptClass, static self => self.BindScriptClass(), this);
```

Reply:
My testing didn't hit much usage of this, will go ahead and change to StrongBox and the compiler folks can chime in if they prefer the original change